LoRaEngine/deployment.test.template.json

```diff
@@ -98,6 +98,9 @@
               },
               "LOG_TO_UDP_ADDRESS": {
                 "value": "$NET_SRV_LOG_TO_UDP_ADDRESS"
+              },
+              "LEAF_DEVICE_PROTOCOL": {
```

Comment:
This change requires you to modify the azure-pipelines.yaml file, defining the value

Reply:
Ok. I'm on it. Sorry, I'm pretty new to the CI pipeline workings

Reply:
@fbeltrao I'm not sure I want to document the MQTT settings is something internal to the engine that should not be changed. We want it as parameter so we can easily test what works best but once we decide we stick with it

Reply:
@fbeltrao from my early test with MQTT if you don't do abandon but also no complete next pass you get the message so it is not an issue but we need to test. for a while we where using MQTT. The issue we had with MQTT is that sometime (1 msg only) will show up on the next pass even after complete is called.....